unity/Assets/Scripts/BaseFPSAgentController.cs

```diff
@@ -2597,6 +2598,79 @@ bool includeInvisible
             actionFinished(true);
         }
 
+        public void Get2DSemanticHulls(List<string> objectTypes = null) {
+            HashSet<string> allowedObjectTypesSet = null;
+            if (objectTypes != null) {
+                allowedObjectTypesSet = new HashSet<string>(objectTypes);
+            }
+
+            var objectIdToConvexHull = new Dictionary<string, List<(float x, float y)>>();
+            foreach (var sop in GameObject.FindObjectsOfType<SimObjPhysics>()) {
+                if (
+                    allowedObjectTypesSet != null 
+                    && !allowedObjectTypesSet.Contains(sop.Type.ToString())
+                ) {
```

Comment:
Debatably would make this:
```c#
if (allowedObjectTypesSet != null &&
    !allowedObjectTypesSet.Contains(sop.Type.ToString())
) {
```
The C# autoformatters will ignore both cases.

Reply:
My style is generally that either everything is on a single line with the parens or the parens are on their own.